### PR TITLE
Home: Swap photo library video for a YouTube video.

### LINKED ISSUE #39910: Home: Free Images Video is Too Fast
As reported by @apeatling  the video for “Over 40,000 Free Photos” is really really fast, and makes it a bit difficult to follow.

Let's try and slow the video to half speed, and let the viewer orient themselves before the video starts navigation. This might be a fade in for a few seconds, or simply pausing for a few seconds before the video plays.

This is also an opportunity to move this video to VideoPress like in https://github.com/Automattic/wp-calypso/blob/f2c9d99b07e14ea938829e9b2e5b31b6fb388a90/client/my-sites/feature-upsell/ads-upsell.jsx#L75

This was previously discussed in p4TIVU-9n1-p2 that we may want to keep video assets out of the codebase to keep the repository size down.

### PR DESCRIPTION
**Pull request by kwight**
Swap out the existing photo library mp4 for a slower-paced video embedded with YouTube.

Mar-13-2020 16-18-04

#### Testing instructions

* In developer console, type the following: `localStorage.debug='calypso:analytics*'`
* Go to My Home (`/home/:site`) and click the "Over 40,000 free images" graphic in the sidebar.
* Verify the video appears in YouTube chrome and works correctly.
* Verify that the following tracks events fire
Screen Shot 2020-03-13 at 4 55 21 PM
* Click on Learn more
* Verify that we're taken to the correct support page, and that the following tracks events fires
Screen Shot 2020-03-13 at 4 55 46 PM

Fixes #39910

### CODE REVIEW

**Comment by mmtr** [comment]
Thanks @kwight! Video is indeed easier to follow now.

Is there any way we can keep the existing autoplay behavior when the modal opens?

**Comment by jancavan** [comment]
Working as expected! I think it isn't great that we show a video without any audio at all, but I'm glad we're replacing that.

**Comment by gwwar** [comment]
If folks wait a bit, we should have a YouTube video available shortly.

**Comment by noahtallen** [comment]
I'm assuming all those extra pings were a result of those extra git commits that slid into the branch ;)

**Comment by kwight** [comment]
> Is there any way we can keep the existing autoplay behavior when the modal opens?

@mmtr Yep! But Chrome et al. might block it anyways 🙃 

Screen Shot 2020-03-12 at 3 38 10 PM

**Comment by Aurorum** [comment]
Should be possible to use the YouTube video now: https://youtu.be/DEnhdAfrHw0

**Comment by gwwar** [comment]
@Aurorum if you're interested would you like to spin up an alternative PR with the YouTube embed? If not just let us know 👍 

One thing to watch out for is adding a `?rel=0` query param so any suggested videos stay with the Automattic channel. Unfortunately YouTube removed the option to disable video suggestions a while back.

**Comment by gwwar** [comment]
@Aurorum I'll go ahead and update this branch for now to see how the YouTube video behaves 😄

**Comment by gwwar** [comment]
Updated to the new YouTube video. Thanks again to @Aurorum who made the original one! A couple of questions for folks:

- Should we still not show the video to mobile devices?
- Do we need a new thumbnail?
- Do the current iframe dimensions look okay?

**Comment by jancavan** [comment]
> Should we still not show the video to mobile devices?

Why weren't we showing it before?

> Do we need a new thumbnail?

You mean this thumbnail, right?
Screen Shot 2020-03-13 at 4 31 29 PM

Yeah, I think something that showcases great photography would be better instead of the current one that's blurred out. I can help with that next week.

> Do the current iframe dimensions look okay?

It's fine, although is it possible for it to be bigger in larger screens?

**Comment by gwwar** [comment]
> Why weren't we showing it before?

@jancavan  We had previously embedded the full video asset, so folks were concerned about the asset size. I do think YouTube would be better optimized for this.

> You mean this thumbnail, right?

Yes!

> It's fine, although is it possible for it to be bigger in larger screens?

@mmtr or @kwight feel free to experiment a bit. The iframe dimensions here were the suggested ones from YouTube, but we might be able to come up with something more responsive.

**Comment by gwwar** [comment]
👍 added some missing tracks events. Let me know if stats or ga would also be useful. 

**Bonus question**: Should learn more open the support page in a new tab. (We'd of course add the external icon link to the button).

**Comment by lancewillett** [comment]
In response to @gwwar's bonus question:

> Should learn more open the support page in a new tab? (We'd of course add the external icon link to the button).

The principle to keep in mind for WordPress.com hosted help articles — meaning written by Automattic on our support sites — is to keep the entire experience within the Calypso app context.

Please use the mechanism that already exists to show the article in an overlay first. That overlay could then link to open the full help article in a new tab/window.

Steps to repeat this, current behavior:

1. Log in, go to Customer Home
2. Click the `?` help icon, bottom right
3. Click "Quick-Start Guide" — you'll see a "Read More" link as a blue button
4. Click the blue button to see the overlay
5. Bottom of the overlay you'll see the external article link

Screenshot for step 4:
Screen Shot 2020-03-14 at 10 26 16

Screenshot for step 5:
Screen Shot 2020-03-14 at 10 26 24

**Comment by Aurorum** [comment]
@lancewillett Just out of curiosity, would that also apply to #39617? The dialog there contains the button and a video, so I'm not sure how that would work. 

Screenshot 2020-02-26 at 17 03 10

**Comment by lancewillett** [comment]
@Aurorum Yes, the same principle apply there. Good call — the idea is to not spawn a bunch of new tabs or windows during a session on WordPress.com, keeping the experience in the same place as much as possible. 

In the case of a help article, it also makes it more likely the person can get right back to the context at hand.

**Comment by Aurorum** [comment]
@gwwar I think you've got the YouTube video under control, but I've created another PR for the support article issue mentioned above: #40157 :)

**Comment by mmtr** [comment]
> > It's fine, although is it possible for it to be bigger in larger screens?
> 
> @mmtr or @kwight feel free to experiment a bit. The iframe dimensions here were the suggested ones from YouTube, but we might be able to come up with something more responsive.

Pushed a commit that adds responsiveness support:

Mar-16-2020 14-06-58

File: client/my-sites/customer-home/free-photo-library-card/style.scss [hunk 1]
```diff
@@ -18,9 +18,20 @@
 .free-photo-library-card__demonstration-video {
 	max-height: 600px;
 	max-width: 800px;
-	width: 100%;
-	
-	&:focus {
-		outline: none;
+	width: calc( 90vw - 16px * 2 );
```
Comment (mmtr) [inline client/my-sites/customer-home/free-photo-library-card/style.scss]: For folks reviewing:
- `90vw`: [Dialog width](https://github.com/Automattic/wp-calypso/blob/146f5ae230fb3e966fd9d53949a55447390b4c8b/packages/components/src/dialog/style.scss#L35).
- `16px`:  [Dialog padding in mobile screens](https://github.com/Automattic/wp-calypso/blob/146f5ae230fb3e966fd9d53949a55447390b4c8b/packages/components/src/dialog/style.scss#L44).

File: client/my-sites/customer-home/free-photo-library-card/style.scss [hunk 2]
```diff
@@ -18,9 +18,20 @@
...
+	position: relative;
+	padding-bottom: calc( 9 / 16 * 100% );
```
Comment (mmtr) [inline client/my-sites/customer-home/free-photo-library-card/style.scss]: `9 / 16`: Inverse of the video aspect ratio (16 / 9).

File: client/my-sites/customer-home/free-photo-library-card/style.scss [hunk 3]
```diff
@@ -18,9 +18,20 @@
...
+	height: 0;
+
+	@include breakpoint( '>480px' ) {
+		width: calc( 90vw - 24px * 2 );
```
Comment (mmtr) [inline client/my-sites/customer-home/free-photo-library-card/style.scss]: - `90vw`: [Dialog width](https://github.com/Automattic/wp-calypso/blob/146f5ae230fb3e966fd9d53949a55447390b4c8b/packages/components/src/dialog/style.scss#L35).
- `24px`:  [Dialog padding in bigger screens](https://github.com/Automattic/wp-calypso/blob/146f5ae230fb3e966fd9d53949a55447390b4c8b/packages/components/src/dialog/style.scss#L48).

**Review by mmtr — APPROVED** [review]
Thanks @gwwar! This tests well for me but I'd appreciate if you could also verify my responsiveness changes before landing.

**Comment by sixhours** [comment]
@mmtr The responsive behavior works well for me! 👍

**Comment by gwwar** [comment]
LGTM! @mmtr you're welcome to land this if y'all like changes here.

**Pull request merged**